app/models/fullcalendar_engine/event.rb

```diff
@@ -17,8 +17,10 @@ class Event < ActiveRecord::Base
     }
     
     def validate_timings
-      if (starttime >= endtime) and !all_day
-        errors[:base] << "Start Time must be less than End Time"
+      if starttime.present? and endtime.present?
+        if (starttime >= endtime) and !all_day
+          errors[:base] << "Start Time must be less than End Time"
```

Comment:
I agree that this is not the part of your change, but can you move this error message to `.yml` file and use it here.